conjure-java-undertow-runtime/src/main/java/com/palantir/conjure/java/undertow/runtime/ConjureHandler.java

```diff
@@ -186,4 +221,8 @@ private void checkOverlappingPaths() {
             }
         }
     }
+
+    private static String normalizeTemplate(String template) {
+        return template.replaceAll("\\{.*?\\}", "{param}");
```

Comment:
Can we include a test to prove this works with multiple path parameters? I _think_ that `.*?` is non-greedy, but I never trust myself to eyeball regexes and not miss something

Reply:
👍